eng/scripts/Detect-ApiChanges.ps1

```diff
@@ -0,0 +1,129 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [array] $ArtifactList,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $APIViewUri,
+  [Parameter(Mandatory=$True)]
+  [string] $buildId,
+  [Parameter(Mandatory=$True)]
+  [string] $repoName,
+  [Parameter(Mandatory=$True)]
+  [string] $PullRequestNumber,
+  [Parameter(Mandatory=$True)]
+  [string] $CommitSha,
+  [string] $TargetBranch = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/")
+)
+
```

Comment:
Lets try to add `Set-StrictMode -version 3` here.

Reply:
I will add it in common script